lib/rom/sql/schema/column_inferrer.rb

```diff
@@ -0,0 +1,82 @@
+require 'rom/sql/types/pg'
```

Comment:
@jodosha well, technically, this is a manageable thing to do—to load a proper extension based on what type of connection we've got, though it'll require some rework around the inferrer infrastructure. /cc @solnic